HOWTO

```diff
@@ -1072,10 +1072,13 @@ I/O type
 
 .. option:: offset=int
 
-	Start I/O at the given offset in the file. The data before the given offset
-	will not be touched. This effectively caps the file size at `real_size -
-	offset`. Can be combined with :option:`size` to constrain the start and
-	end range that I/O will be done within.
+	Start I/O at the provided offset in the file, given as either a fixed size or
+	a percentage. The data before the given offset will not be touched. This
+	effectively caps the file size at `real_size - offset`. Can be combined with
+	:option:`size` to constrain the start and end range that I/O will be done
+	within.  It is also possible to give offset as a percentage between 1 and
+	100. If ``offset=20%`` is given, fio will skip the first 20% of the file and
+	begin I/O.
```

Comment:
Perhaps mention "(rounding up to the nearest block if necessary)"? What do you think?